src/Themes/HackathonTheme/Views/Widget-VInput.liquid

```diff
@@ -0,0 +1,31 @@
+{% assign model = Model.ContentItem.Content.VInput.Model.Text %}
```

Comment:
Should this be part of the VueForms module ?

Reply:
Probably makes more sense in the VueForms module, for sure.